Python.gitignore

```diff
@@ -108,6 +108,7 @@ ipython_config.py
 #   in version control.
 #   https://pdm.fming.dev/#use-with-ide
 .pdm.toml
```

Comment:
They also renamed their config file from `.pdm.toml` to `pdm.toml`
Maybe both two names should be at here for backward compatibility?

Reply:
```suggestion
.pdm.toml
pdm.toml
```

Reply:
Quoting the release notes:

> Move the project python path to its own file, and rename the project config file as pdm.toml which can be committed to the VCS. https://github.com/pdm-project/pdm/issues/1742

As far as I understand this, the sole purpose of `.pdm-python` is to remove environment specific configuration from the old `.pdm.toml`, creating a newly named `pdm.toml` which can be commited to VCS, so it should not be ignored by default anymore.